packages/stepper/src/styledComponents/Step.js

```diff
@@ -0,0 +1,52 @@
+import styled, { css } from "styled-components";
+
+import colorTypes, { CARBON } from "../constants/colorTypes";
+import statusTypes from "../constants/statusTypes";
+
+const StepContainer = styled.div`
+  font-family: Lato, sans-serif;
+  size: 18px;
+  line-height: 1.33;
+
+  display: flex;
+
+  border-radius: 4px;
```

Comment:
I think that it's a good idea to start thinking on a way to convert `pixel` values to `rem`